build.sbt

```diff
@@ -51,7 +51,7 @@ ThisBuild / scmInfo ~= {
 // Global settings.
 ThisBuild / scalaVersion := "2.12.14"
 ThisBuild / crossScalaVersions := Seq("2.12.14", "2.13.6")
-ThisBuild / organization := "com.dimafeng"
+ThisBuild / organization := "io.github.neotypes"
```

Comment:
Do we also need to make some changes to [the `publishTo` settings](https://github.com/neotypes/neotypes/pull/359/files#diff-5634c415cd8c8504fdb973a3ed092300b43c4b8fc1e184f7249eb29a55511f91R73-R83)?

Reply:
For some reason your link is not working. I'll figure out what else needs to be changed during release